eng/Versions.props

```diff
@@ -10,6 +10,8 @@
     <VersionSDKMinor>1</VersionSDKMinor>
     <VersionFeature>00</VersionFeature>
     <VersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)$(VersionFeature)</VersionPrefix>
+    <!-- DO NOT CHANGE - The feature band version is critical for the finalizer and must never include the SDK patch level -->
+    <FeatureBandVersion>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)00</FeatureBandVersion>
```

Comment:
Do we need to define this value separately?  We end up calling `ParseSdkVersion` on it anyway, so would it be simpler to just pass in the actual full SDK version to the finalizer?

Reply:
Yeah, Now that we have the parsing ability we should. I'll update the PR

Reply:
I updated the PR. When we call the finalizer now, we can just pass in the log, full SDK version and platform. The finalizer will construct the dependent value and the target SDK feature band version.

FYI, it seems in the SDK we're using a property called NuGetVersion for the SDK version. I take it that's some leftover artifact from long ago when we had the CLI repo